pynest/nest/lib/hl_api_simulation.py

```diff
@@ -382,29 +382,31 @@ def SetKernelStatus(params):
     GetKernelStatus
 
     """
-    # Read docstring
-    lines = SetKernelStatus.__doc__.split('\n')
-
-    # Get the lines describing parameters, excluding the first one
-    param_lines = [line for line in lines if ' : ' in line][1:]
-
-    # Test if the provided parameters are valid and whether they can be set
     for key in params.keys():
-        keyline = [line for line in param_lines if key + ' : ' in line]
-        if len(keyline) == 0:
+        readonly = _sks_params.get(key)
```

Comment:
This would also have to change to `readonly = SetKernelStatus._sks_params.get(key)`, see the other comment